modules/planning/common/frame.cc

```diff
@@ -195,29 +195,80 @@ bool Frame::CreateReferenceLineInfo() {
  * @brief: create static virtual object with lane width,
  *         mainly used for virtual stop wall
  */
-const Obstacle *Frame::AddVirtualStopObstacle(
-    ReferenceLineInfo *const reference_line_info, const std::string &object_id,
-    const double object_s) {
+const Obstacle *Frame::CreateVirtualStopObstacle(
+    ReferenceLineInfo *const reference_line_info,
+    const std::string &obstacle_id,
+    const double obstacle_s) {
   if (reference_line_info == nullptr) {
     AERROR << "reference_line_info nullptr";
     return nullptr;
   }
 
   const auto &reference_line = reference_line_info->reference_line();
-  const double box_center_s = object_s + FLAGS_virtual_stop_wall_length / 2.0;
+  const double box_center_s = obstacle_s + FLAGS_virtual_stop_wall_length / 2.0;
   auto box_center = reference_line.GetReferencePoint(box_center_s);
-  double heading = reference_line.GetReferencePoint(object_s).heading();
+  double heading = reference_line.GetReferencePoint(obstacle_s).heading();
   double lane_left_width = 0.0;
   double lane_right_width = 0.0;
-  reference_line.GetLaneWidth(object_s, &lane_left_width, &lane_right_width);
+  reference_line.GetLaneWidth(obstacle_s, &lane_left_width, &lane_right_width);
   Box2d stop_wall_box{box_center, heading, FLAGS_virtual_stop_wall_length,
                       lane_left_width + lane_right_width};
 
-  return AddStaticVirtualObstacle(object_id, stop_wall_box);
+  return CreateStaticVirtualObstacle(obstacle_id, stop_wall_box);
 }
 
-const Obstacle *Frame::AddStaticVirtualObstacle(const std::string &id,
-                                                const Box2d &box) {
+/**
+ * @brief: create static virtual object with lane width,
+ */
+const Obstacle *Frame::CreateVirtualStaticObstacle(
```

Comment:
make function names consistent.   CreateStateVirtualObstacle at line 217, but another function called CreateVirutalStaticObstacle at line 223.

Reply:
Changed CreateStateVirtualObstacle() at line 217 to CreateStateObstacle(). This is a public method used to create a static obstacle. 

CreateVirutalStaticObstacle() at line 223 is a private method used to by all obstacle creation method (stop wall,  static obstacle and etc).  The name specify virtual + static.